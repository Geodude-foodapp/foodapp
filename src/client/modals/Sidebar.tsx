import React from 'react';

type SidebarProps = {
  closeSidebar: () => void;
};
export default ({ closeSidebar }: SidebarProps) => {
  return (
    <>
      <h1>Sidebar</h1>
      <button onClick={closeSidebar}>Close Sidebar</button>
    </>
  );
};
