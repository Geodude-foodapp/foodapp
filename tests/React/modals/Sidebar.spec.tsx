import { render } from '@testing-library/react';
import React from 'react';
import Sidebar from '../../../src/client/modals/Sidebar';

describe('Recipe Card Component', () => {
  const closeSidebar = jest.fn();
  beforeAll(() => render(<Sidebar closeSidebar={closeSidebar} />));
  it('matches the snapshot', () => {
    const sidebar = document.querySelector('#sidebar');
    expect(sidebar).toMatchSnapshot();
  });
});
