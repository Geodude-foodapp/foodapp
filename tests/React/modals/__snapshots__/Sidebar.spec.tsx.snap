// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe Card Component matches the snapshot 1`] = `
<section
  id="sidebar"
>
  <h1>
    Sidebar
  </h1>
  <button>
    Close Sidebar
  </button>
</section>
`;
